src/view-data/method.ts

```diff
@@ -113,9 +113,13 @@ function getPathToMethodName(httpVerb: string, path: string): string {
   });
 
   const result = camelCase(segments.join("-"));
-  return `${httpVerb.toLowerCase()}${result[0].toUpperCase()}${result.substring(
-    1
-  )}`;
+  if (result.length > 0) {
+    return `${httpVerb.toLowerCase()}${result[0].toUpperCase()}${result.substring(
+      1
+    )}`;
+  }
+  // handle case when "/" is passed as a path
+  return `${httpVerb.toLowerCase()}_root_`;
```

Comment:
>_root_ [](start = 35, length = 6)

There is the edge case where an endpoint is actually called "/_root_" (or whatever name we decide on), where this would cause a collision. We can either add logic to check for this in the if branch above, or we can change up the format of the root name to something like "rootEndpoint_get", which ofc. breaks consistency

Reply:
Done, decided to break the convention